ci-operator/step-registry/service-registry/run-tests/service-registry-run-tests-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Run the tests"
+./scripts/run-interop-tests.sh
+
+echo "Copy logs and xunit to artifacts dir"
+cp target/surefire-reports/*.xml "${ARTIFACT_DIR}"
```

Comment:
We need to also make sure that the XML files start with `junit_`. This is needed for the test cases to appear on the TestGrid Dashboard for this scenario.

The XMLs generated are [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/38858/rehearse-38858-periodic-ci-Apicurio-apicurio-registry-system-tests-main-service-registry-ocp4.13-lp-interop-service-registry-2-3/1658725554989305856/artifacts/service-registry-2-3/service-registry-run-tests/artifacts/) these just need to read for example.. `junit_TEST-io.apicurio.registry.systemtests.api.BundleAPITests.xml`

using something like this should work

```
rename TEST junit_TEST /logs/artifacts/TEST*.xml
```

Reply:
This will also be required to report [Jira bugs on test failures when the project is done](https://issues.redhat.com/browse/LPTOCPCI-138).